modules/rabbitmq/collect.go

```diff
@@ -116,10 +117,18 @@ func (r *RabbitMQ) parse(mx *metrics) map[string]int64 {
 	if mx.hasVhostsStats() {
 		for _, vhost := range mx.vhosts {
 			for k, v := range stm.ToMap(vhost) {
-				ms[fmt.Sprintf("vhost_%s_%s", vhost.Name, k)] = v
+				name := cleanVhostName(vhost.Name)
+				ms[fmt.Sprintf("vhost_%s_%s", name, k)] = v
 			}
 		}
 	}
 
 	return ms
 }
+
+func cleanVhostName(name string) string {
+	if name == "/" {
+		return name
+	}
+	return strings.TrimPrefix(name, "/")
```

Comment:
vhost may be /foo/bar

Reply:
we should allow / in vhost name but don’t use it in chart id

Reply:
So whats your suggestion?

I think it would be better to have `vhost /foo/bar` family, then `foo_bar`? Do we really need to change vhost name?

Reply:
> we should allow / in vhost name but don’t use it in chart id

what about dimID?